app/models/maintenance_stats/stats/github/issue_stats.rb

```diff
@@ -0,0 +1,64 @@
+module MaintenanceStats
```

Comment:
Could we add a spec for this? The other stats have simple specs that check if the data is in the right spots, https://github.com/librariesio/libraries.io/blob/master/spec/models/maintenance_stats/github/commit_count_stats_spec.rb